lib/tests/group.rs

```diff
@@ -288,8 +288,8 @@ async fn select_aggregate() -> Result<(), Error> {
 #[tokio::test]
 async fn select_multi_aggregate() -> Result<(), Error> {
 	let sql = "
-		CREATE test:1 SET group = 1, one = 1.7, two = 2.4;
-		CREATE test:2 SET group = 1, one = 4.7, two = 3.9;
+		CREATE test:1 SET group = 1, one = 1, two = 2.4;
```

Comment:
Why did these tests change? They should remain as they were

Reply:
Sure, i reverted them.